src/util.c

```diff
@@ -627,9 +627,67 @@ int dcc_which(const char *command, char **out)
     }
 }
 
-/* Returns the number of processes in state D, the max non-cc/c++ RSS in kb and
- * the max RSS program's name */
-void dcc_get_proc_stats(int *num_D, int *max_RSS, char **max_RSS_name) {
+/* Returns the value from the "MemAvailable" line in the opened file, converted
```

Comment:
I guess this will compile anywhere but please wrap it in `#ifdef linux` for clarity?